WD-Firmware/board/nexys_a7/bsp/env/freedom-e300-hifive1/init.c

```diff
@@ -0,0 +1,256 @@
+#include <stdint.h>
```

Comment:
remove/rename the folder freedom-e300-hifive1

Reply:
[will be done after new main.c]

Reply:
Done by Ofer